### DISCUSSION TITLE
Use mitosis to create multiple SDKs

### DISCUSSION THREAD

**Discussion by AlgorithmEnigma** [clerk/javascript#1362] [Feature Requests]
If you create the library using [Mitosis](https://github.com/BuilderIO/mitosis) you would be able to have SDKs/Component Libraries for a wide range of frameworks, allowing you to use clerk easier with non-react/nextjs projects.

**Comment by jescalan** [comment]
Thank you for the suggestion! We will certainly discuss this 😁
